Directory.Packages.props

```diff
@@ -7,7 +7,7 @@
     <RoslynVersion>4.13.0</RoslynVersion>
     <RoslynVersionForAnalyzers>4.11.0</RoslynVersionForAnalyzers>
     <CodeAnalysisAnalyzerVersion>3.11.0-beta1.24629.2</CodeAnalysisAnalyzerVersion>
-    <PolyTypeVersion>0.29.3</PolyTypeVersion>
+    <PolyTypeVersion>0.29.3-g4b0aeb5295</PolyTypeVersion>
```

Comment:
This is currently blocked on the merge of https://github.com/eiriktsarpalis/PolyType/pull/128